kardinal-manager/kardinal-manager/cluster_manager/cluster_manager.go

```diff
@@ -284,6 +284,12 @@ func (manager *ClusterManager) CleanUpClusterResources(ctx context.Context, clus
 	servicesByNS := lo.GroupBy(*clusterResources.Services, func(item corev1.Service) string {
```

Comment:
This code seems to support multiple namespaces but we only support one during deploy.  I will sync up with the team and see what we need to keep here.